BrainPortal/lib/cbrain_task_generators/schema_task_generator.rb

```diff
@@ -166,6 +165,12 @@ def integrate(register: true, create_tool_config: false, multi_version: false)
         })[partial]
       end
 
+      # Add a helper method for accessing the help file (for use on the tools page)
+      # Written by the rake task cbrain:plugins:install, within the subtask public_assets
+      helpFileName = name + "_help.html" 
+      helpFileDir  = File.join( "cbrain_plugins", "cbrain_tasks", "help_files/" )
```

Comment:
I don't get this... what is this path "cbrain_plugins/cbrain_tasks" ? It doesn't correspond to the plugins structure?